saito-rust/src/main.rs

```diff
@@ -589,6 +592,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         tokio::sync::mpsc::channel::<MiningEvent>(channel_size);
     let (sender_to_stat, receiver_for_stat) = tokio::sync::mpsc::channel::<String>(channel_size);
 
+    info!("run_verification_threads");    
```

Comment:
these logs don't mean much as thread won't exactly run at this point. correct location is on_init functions in related threads.

Reply:
ok. it would be useful to have high level information to understand whats going on. maybe some status report which can be pulled? i.e. what threads are running and whats the status of the network.

Reply:
so there's a stat file generated on data/saito.stat ( have to cargo run --features with-stats). I haven't documented these yet since rust project wasn't that involved with the community. will try to update the documents when I get some free time.